src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -20,19 +27,55 @@ export const ChannelClaimTableRow = ({ claimedChannel }: ChannelClaimTableRowPro
   const organizationQuery = useFetchOrganization(claimedChannel.claimedBy.organizationId);
   const organizationName = getLanguageString(organizationQuery.data?.labels);
 
+  const publishingPolicy = claimedChannel.channelClaim.constraint.publishingPolicy;
+  const editingPolicy = claimedChannel.channelClaim.constraint.editingPolicy;
+
   return (
-    <TableRow>
+    <TableRow sx={{ bgcolor: 'white' }}>
+      <StyledTableCell aria-live="polite" aria-busy={publisherQuery.isPending}>
+        {publisherQuery.isPending ? (
+          <Skeleton width={300} />
+        ) : publisherName ? (
+          <Typography>{publisherName}</Typography>
+        ) : (
+          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+        )}
+      </StyledTableCell>
+      <StyledTableCell aria-live="polite" aria-busy={organizationQuery.isPending}>
+        {organizationQuery.isPending ? (
+          <Skeleton width={300} />
+        ) : organizationName ? (
+          <Typography>{organizationName}</Typography>
+        ) : (
+          <Typography sx={{ fontStyle: 'italic' }}>{t('common.unknown')}</Typography>
+        )}
+      </StyledTableCell>
       <StyledTableCell>
-        {publisherName ? publisherName : <span style={{ fontStyle: 'italic' }}>{t('common.unknown')}</span>}
+        <StyledChip
+          size="small"
+          sx={{
+            bgcolor: publishingPolicy === 'Everyone' ? 'publishingRequest.main' : 'centralImport.main',
+          }}
+          label={t(`editor.institution.channel_claims.access_policies.${publishingPolicy}`)}
+          icon={
+            publishingPolicy === 'Everyone' ? <LockOpenIcon fontSize="small" /> : <LockOutlineIcon fontSize="small" />
+          }
+        />
       </StyledTableCell>
       <StyledTableCell>
-        {organizationName ? organizationName : <span style={{ fontStyle: 'italic' }}>{t('common.unknown')}</span>}
+        <Chip
```

Comment:
Skal ikke denne også være Styled på samme måte som publishing policy?
```suggestion
        <StyledChip
```